web/third/bootstrap-typeahead/typeahead.js

```diff
@@ -137,13 +137,13 @@ import {insertTextIntoField} from "text-field-edit";
 import {get_string_diff} from "../../src/util";
 
 !(function ($) {
-    function get_pseudo_keycode(e) {
+    function get_pseudo_keycode(event) {
         const isComposing = (event.originalEvent && event.originalEvent.isComposing) || false;
         /* We treat IME compose enter keypresses as a separate -13 key. */
-        if (e.keyCode === 13 && isComposing) {
+        if (event.keyCode === 13 && isComposing) {
             return -13;
         }
-        return e.keyCode;
+        return event.keyCode;
```

Comment:
This looks like it just renames a variable?

Reply:
No, it fixes the existing references to `event` that had already been added to this function (on the lines that aren’t being touched here), and previously referred unintentionally to the deprecated global `window.event`.

Reply:
I'll update this to not rename the variable in the same commit